src/golang/lib/repos/tests/seed.go

```diff
@@ -1,6 +1,28 @@
 package tests
 
-import "github.com/aqueducthq/aqueduct/lib/models"
+import (
+	"github.com/aqueducthq/aqueduct/lib/models"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// Defaults used for seeding database records
+	testOrgID = "aqueduct-test"
+)
+
+// seedUser creates count user records.
```

Comment:
Nit: Put backticks around count to make it more explicit it is a variable.
e.g. // seedUser creates `count` user records.